src/main/java/core/basesyntax/service/impl/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriterService;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String resultingReport, String filePath) {
+        File file = new File(filePath);
+        try {
+            Files.write(file.toPath(), resultingReport.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file, " + e);
+        }
+    }
```

Comment:
The checklist mentions using try-with-resources for classes that implement the AutoCloseable interface. While Files.write() method handles resource closing internally, it's good to know when you're working with streams directly (e.g., new BufferedWriter(new FileWriter(file))) you would need to ensure resources are closed properly.